lib/iris/common/resolve.py

```diff
@@ -338,11 +357,31 @@ def __call__(self, lhs, rhs):
         return self
 
     def _as_compatible_cubes(self):
+        """
+        Determine whether the ``src`` and ``tgt`` :class:`~iris.cube.Cube` can
+        be transposed and/or broadcast successfully together.
+
+        If compatible, the ``_broadcast_shape`` is calculated of the resultant
+        resolved cube, and the ``_src_cube_resolved`` (transposed/broadcast ``src``
```

Comment:
I find the following wording easier to read
```suggestion
        If compatible, the ``_broadcast_shape`` of the resultant resolved cube is
        calculated, and the ``_src_cube_resolved`` (transposed/broadcast ``src``
```